qa/tasks/deepsea.py

```diff
@@ -1465,7 +1467,28 @@ def begin(self):
                 "config dictionary may contain only one key. "
                 "You provided ->{}<- keys ({})".format(len(config_keys), config_keys)
                 )
-        role_spec, repositories = self.config.items()[0]
+        role_spec, paramdict = self.config.items()[0]
+        if not isinstance(paramdict, dict):
+            paramdict = {'tries': reboot_tries}
+        paramdict_keys = len(paramdict)
+        if paramdict_keys >= 1:
+            self.log.info("Considering parameter dict {}".format(paramdict))
+            if paramdict_keys > 1:
+                raise ConfigError(
+                    self.err_prefix +
+                    "parameter dict may contain only one key. "
+                    "You provided ->{}<- keys".format(paramdict_keys)
+                    )
```

Comment:
Why not to move this check before `paramdict_keys >= 1`?

Reply:
and use later `elif paramdict_keys == 1`

Reply:
Done